src/mgmt/srv_pool.c

```diff
@@ -668,3 +668,87 @@ ds_mgmt_pool_list(void)
 out:
 	return rc;
 }
+
+/* Caller is responsible for freeing ranks */
+static int
+pool_get_ranks(struct mgmt_svc *svc, uuid_t uuid, d_rank_list_t **ranks)
```

Comment:
Im assuming the reason this is in here is because of the PR that Tom mentions above?

Reply:
No, we need the pool ranks in order to send a message over CaRT to the pool service. I didn't see any existing method to get the rank list via the mgmt svc.